spec/traits.dd

```diff
@@ -630,12 +630,35 @@ writeln(__traits(identifier, var));             // "var"
 
 $(H2 $(GNAME getAliasThis))
 
-    $(P Takes one argument, a symbol of aggregate type.
-        If the given aggregate type has $(D alias this), returns a list of
-        $(D alias this) names, by a tuple of $(D string)s.
-        Otherwise returns an empty tuple.
+    $(P Takes one argument, a symbol. If the symbol is of a type which has
+        $(D alias this), returns a list of $(D alias this) names, by a tuple
+        of $(D string)s. Otherwise returns an empty tuple.
     )
 
+$(SPEC_RUNNABLE_EXAMPLE_COMPILE
+---
+alias TypeTuple(T...) = T;
```

Comment:
As mentioned above, `TypeTuple` is the wrong name, as it's neither a tuple, nor in the example below holds any types.

```suggestion
alias AliasSeq(Args...) = Args;
```

Reply:
Implemented.